tendermint/src/lite/error.rs

```diff
@@ -0,0 +1,73 @@
+//! All error types tied to the light client.
+
+use crate::Hash;
+use anomaly::{BoxError, Context};
+use std::time::{SystemTime, SystemTimeError};
+use thiserror::Error;
+
+/// The main error type verification methods will return.
+/// See [`ErrorKind`] for the different kind of errors.
+pub type Error = anomaly::Error<Kind>;
```

Comment:
How is this used? Seems everything is returning Kind

Reply:
As per #151 everything should be using Error and not Kind anymore. (Can you re-approve btw).